docs/src/TapArea.doc.js

```diff
@@ -220,26 +220,37 @@ card(
         defaultValue: 'null',
         description: [
           'Define the frame or window to open the anchor defined on `href`:',
-          '- `null` opens the anchor in the same window.',
-          '- `blank` opens the anchor in a new window.',
-          '- `self` opens an anchor in the same frame.',
+          `- 'null' opens the anchor in the same window.`,
+          `- 'blank' opens the anchor in a new window.`,
+          `- 'self' opens an anchor in the same frame.`,
           'Optional with role=link.',
         ],
         href: 'roles',
       },
+      {
+        name: 'tapStyle',
+        type: `none | 'compress'`,
+        required: false,
+        defaultValue: 'none',
+        description: [
+          'Set a compressing behavior when the TapArea is clicked / touched.',
+          `- 'none' does not compress TapArea.`,
+          `- 'compress' scales down TapArea.`,
+        ],
+        href: 'tapStyle',
```

Comment:
@AlbertCarreras this link doesn't seem to work - should it be `roles` instead?

Reply:
DONE

Reply:
I was going to create it's own example, but add it there at the end.